src/main/java/WorkShop01.kt

```diff
@@ -1,37 +1,64 @@
+import java.lang.Integer.max
+import java.math.BigDecimal
+
 object WorkShop01 {
 
     fun numericValues() {
         //TODO 1.1. Необходимо создать целочисленную переменную, которая может вместить 10^9
 
+        val var1: Int = 1000000000
+
         //TODO 1.2. Необходимо создать целочисленную переменную, которая может вместить 10^10
+
+        val var2: Long = 10000000000
+
     }
 
     fun printRealOnePrecisionValue() {
         //TODO 2.1. Необходимо создать вещественную переменную с одинарной точности и инициализировать её значением 0.0F
 
+        var var1: Float = 0.0F
+
         //TODO 2.2. Добавить к созданной переменной число 0.1111111111111111F 2 раза
 
+        var1 += 0.1111111111111111F.times(2)
+
+
+        val bd1: BigDecimal = BigDecimal(Float.toString(var1))
```

Comment:
```suggestion
        val bd1: BigDecimal = BigDecimal(var1.toString())
```
Не стоит для таких простых вещей делать обертки